CMakeLists.txt

```diff
@@ -1,33 +1,51 @@
 project(cmakesample CXX)
-cmake_minimum_required(VERSION 2.8.11)
+#cmake_minimum_required(VERSION 2.8.11)
```

Comment:
Think we can just assume 3.x getting used already and remove commented out lines like this etc.

Reply:
Ok.